lizmap/modules/lizmap/lib/App/JelixContext.php

```diff
@@ -210,7 +211,14 @@ public function flushCache($profile = '')
      */
     public function logMessage($message, $cat = 'default')
     {
-        \jLog::log($message, $cat);
+        if ($cat == 'lizmapadmin') {
+            \jLog::log($message, $cat);
+            \lizmap::getLogger()->error($message);
+        } elseif (in_array($cat, Logger::LogLevels)) {
+            \lizmap::getLogger()->log($cat, $message);
+        } else {
+            \jLog::log($message, $cat);
+        }
```

Comment:
In a long term, maybe this function should launch a `deprecated` error ? 

We may add a `CustomLogger`  acting as PSR-3 logger but for the `lizadmin` file ?

````
// \lizmap::getAppContext()->logMessage($errormsg, 'lizmapadmin');
lizmap::getCustomLoger()->debug($errormsg);
````